app/components/Navigation.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { connect as connectFela } from 'react-fela'
+import { pipe } from 'ramda'
+import AppBar from 'material-ui/AppBar'
+import Drawer from 'material-ui/Drawer'
+import MenuItem from 'material-ui/MenuItem'
+import Divider from 'material-ui/Divider'
+
+import styles from '../styles/Navigation'
+import { FormattedMessage } from '../../lib/Intl'
+
+class Navigation extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
```

Comment:
i know i mentioned this before, but what do you think about using `recompose` to handle stateful or lifecycle components? while i did say "functional purity" can be a trap, i am enjoying using `recompose` so far.

so this would be:

```js
import { connect as connectFela } from 'react-fela'
import { withState, withHandlers } from 'recompose'
import { pipe, not } from 'ramda'

import styles from '../styles/Navigation'

const Navigation = (props) => {
  // render...
}

export default pipe(
  connectFela(styles),
  withState('isDrawerOpen', 'setDrawerOpen', false),
  withHandlers({
    toggleDrawer: ({ setDrawerOpen }) => () => setDrawerOpen(not)
  })
)(Navigation)
```